helm/charts/hydra-maester/values.yaml

```diff
@@ -49,6 +55,16 @@ deployment:
     #   cpu: 100m
     #   memory: 20Mi
 
+  # -- If you want to mount external volume
+  extraVolumes: []
```

Comment:
Please update the values in `hacks/values`, so that the CI will use the new fields. We can use bogus/tests values there, just so the new fields are rendered in the CI :)